src/tests/routes/crates/snapshots/crates_io__tests__routes__crates__read__include_default_version.snap

```diff
@@ -0,0 +1,78 @@
+---
+source: src/tests/routes/crates/read.rs
+expression: response.json()
+snapshot_kind: text
+---
+{
+  "categories": null,
+  "crate": {
+    "badges": [],
+    "categories": null,
+    "created_at": "[datetime]",
+    "default_version": "0.5.1",
+    "description": "description",
+    "documentation": "https://example.com",
+    "downloads": 20,
+    "exact_match": false,
+    "homepage": "http://example.com",
+    "id": "foo_default_version",
+    "keywords": null,
+    "links": {
+      "owner_team": "/api/v1/crates/foo_default_version/owner_team",
+      "owner_user": "/api/v1/crates/foo_default_version/owner_user",
+      "owners": "/api/v1/crates/foo_default_version/owners",
+      "reverse_dependencies": "/api/v1/crates/foo_default_version/reverse_dependencies",
+      "version_downloads": "/api/v1/crates/foo_default_version/downloads",
+      "versions": "/api/v1/crates/foo_default_version/versions"
+    },
+    "max_stable_version": null,
+    "max_version": "0.0.0",
+    "name": "foo_default_version",
+    "newest_version": "0.0.0",
+    "recent_downloads": null,
+    "repository": null,
+    "updated_at": "[datetime]",
+    "versions": null,
+    "yanked": false
+  },
+  "default_version": {
```

Comment:
I'm wondering whether we should add this to the `versions` array instead. if both includes are set we wouldn't be duplicating the information in the response, and I think e.g. Ember Data would expect it to be in `versions` instead of a somewhat random other key 🤔

Reply:
Yeah, this is quite interesting. I haven't played around with this part on the frontend yet. I'm guessing we have  versions relationship with `@hasMany` and `default_version` relationship with `@belongsTo`. But indeed, the `default_version` is definitely one of the `versions`.

So, are you suggesting we just share the same key here? Which means that with only `default_version` in the `include` query, the output would be similar to:

```json
{
"crate": {...},
"categories": null,
"keyword": null,
"versions": [{...default_version...}] 
}
```

And if both `versions` and `default_version` are included (or `full`), then we simply treat it as the `versions` and ignore the `default_version`?

Reply:
yeah, exactly